include/wx/fontutil.h

```diff
@@ -118,11 +118,11 @@ class WXDLLIMPEXP_CORE wxNativeFontInfo
     // set the XFLD
     void SetXFontName(const wxString& xFontName);
 #elif defined(__WXMSW__)
-    wxNativeFontInfo(const LOGFONT& lf_)
-        : lf(lf_),
-          pointSize(GetPointSizeAtPPI(lf.lfHeight))
-    {
-    }
+    wxNativeFontInfo(const LOGFONT& lf_, const wxWindow* win
+#ifndef WXBUILDING
```

Comment:
I think this is sufficiently non-trivial to merit an explanation, i.e. something like
```
// Preserve compatibility in the semi-public (i.e. private, but still
// unfortunately used by some existing code outside of the library) API
// by allowing to create wxNativeFontInfo from just LOGFONT, but ensure
// that we always specify the window, to use the correct DPI, when creating
// fonts inside the library itself.
```